utils/gravityUtils.js

```diff
@@ -479,6 +483,35 @@ const getCurrentJupiterValueOrNull = async function (){
     return null
 }
 
+/**
+ *
+ * @param jupAddress
+ * @param alias
+ * @param req
+ * @return {newAccountIp}
+ */
+const ipLogger = function (jupAddress, alias, req) {
+    const ipAddress = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
+    const newAccountIp = {
+        ipAddress: ipAddress,
+        jupAddress: jupAddress,
+        alias: alias,
+        timestamp: new Date()
+    };
+    NewAccountIp.create(newAccountIp);
```

Comment:
it would be nice if we add some logs here in order to monitors if the sign up is coming from a specific IP address